modules/fb_instant_articles_display/src/DrupalInstantArticleDisplay.php

```diff
@@ -62,7 +62,7 @@ class DrupalInstantArticleDisplay {
    *
    * @var InstantArticle
    */
-  private $instantArticle;
+  public $instantArticle;
```

Comment:
Just curious, I see it was part of 97fff38 but why change $instantArticle from private to public?